site/content/extend/plugins/best-practices.md

```diff
@@ -75,26 +75,45 @@ For examples of custom settings see: Demo Plugin [`CustomSetting`](https://githu
 
 ## How can I review the entire code base of a plugin?
 
-Carry out the following steps:
+Sometimes, you have been working on a personal repository for a new plugin, most probably based on the [mattermost-plugin-starter-template](https://github.com/mattermost/mattermost-plugin-starter-template/) repo. As it was a personal project, you may have pushed all of your commits directly commits to `master`. And now that it's functional, you need a reviewer to take a look at the whole thing.
 
-1. Take a backup of project directory
-2. Create a dummy-master branch with no code:
+For this, it is useful to create a PR with only the commits you added. Follow these steps to do so:
+
+0. Make sure that you are in the `master` branch, and that the remote is updated.
```

Comment:
nit: I know the numbering is ultimately re-created, but it's weird to read the raw data and see a `0.` for a list of steps.

Reply:
Done in https://github.com/mattermost/mattermost-developer-documentation/pull/652/commits/c8ab06a7648c27f059aff8b68459af275a64d263.